tests/PictureGeneratorTest.php

```diff
@@ -240,6 +253,125 @@ function () {
         $this->assertInstanceOf('Contao\Image\PictureInterface', $picture);
     }
 
+    public function testGenerateWDescriptorSmallImage()
+    {
+        $resizer = $this->createMock(ResizerInterface::class);
+
+        $resizer
+            ->method('resize')
+            ->will($this->returnCallback(
+                function () {
+                    /** @var ResizeConfigurationInterface $config */
+                    $config = func_get_arg(1);
```

Comment:
O RLY? 😄

Reply:
Yeah, It’s a copy paste from the previous method :D

Reply:
Done in 83dd8d43100a30b491f46a844de2d3263127564d